app/javascript/menu/main-menu.jsx

```diff
@@ -52,7 +52,7 @@ export const MainMenu = ({ applianceName, currentGroup, currentUser, customBrand
 
   return (
     <>
-      <div onClick={hideSecondary}>
+      <div onClick={hideSecondary} id="main-menu">
```

Comment:
id="main-menu" is already existing in page, it is in app/views/layouts/_header.html.haml. Duplicate ids in same html are not reccomended.

Reply:
My bad, I'll try to find a better name :thinking: but this IS the main menu :laughing: